### PR TITLE
Translating home page

### PR DESCRIPTION
**Pull request by fawzisf**

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ class Home extends Component {
     return (
       <Layout location={location}>
         <TitleAndMetaTags
-          title="React &ndash; A JavaScript library for building user interfaces"
+          title="React &ndash; gj', لتطوير واجهات الاستخدام مكبة جافاسكريبت "
```
[the remark this answers is not in the document]
Comment (fawzisf) [reply src/pages/index.js]: so i should include both english and arabic for tech terms ?

[the remark this answers is not in the document]
Comment (22mahmoud) [reply src/pages/index.js]: Yes

Comment (RamyAlshurafa) [inline src/pages/index.js]: مكبة => مكتبة

**Comment by asantarissy** [comment]
@fawzisf what's the status on this?

**Comment by fawzisf** [comment]
Will push towards finishing it this week, was a little pressured last week.

**Comment by asantarissy** [comment]
Hello @fawzisf, @iRayan7 created a new [Glossary wiki](https://github.com/reactjs/ar.reactjs.org/wiki/Glossary), please use it as a reference and keep checking it as it's WIP.

Cheers!

**Pull request closed without merging**